Add tooltipText param to TopNav and LeftNav (#2242)
SAP/luigi: docs/navigation-parameters-reference.md

### nodeChangeHook
- **type**: function
- **description**: allows you to invoke and execute a specific function on the global level when a request to navigate to the node occurs. The function receives two node objects as input parameters: the previous node and current node, as described in the configuration.

### defaults.isolateView
- **type**: boolean
- **description**: renders all views in new frames. This setting overrides the same-domain frame reuse.

> - **default**: the parameter **defaults.isolateView** is `false` by default, and you can overwrite it using the **isolateView** value on a single node level.

Author: I add `defaults.tooltipText` to docu

### nodeChangeHook
- **type**: function
- **description**: allows you to invoke and execute a specific function on the global level when a request to navigate to the node occurs. The function receives two node objects as input parameters: the previous node and current node, as described in the configuration.

### defaults.isolateView
- **type**: boolean
- **description**: renders all views in new frames. This setting overrides the same-domain frame reuse.
- **default**: the parameter **defaults.isolateView** is `false` by default, and you can overwrite it using the **isolateView** value on a single node level.

### tooltipText
- **type**: string
- **description**: allows you to add a tooltip to any of the links and buttons of the left side navigation and top navigation. You can set the parameter with the text that will appear on the tooltip.
- **default**: it is `null` by default, and you can overwrite it using the **tooltipText** value on a single node level.
- **example**:
```javascript
tooltipText: 'Useful links'
```

### defaults.tooltipText
- **type**: string

> - **description**: defaults properties apply to nodes where they are not set explicitly, this is just a dumb mechanism.

Reviewer: ```suggestion
- **description**: applies the [tooltipText](navigation-parameters-reference.md#tooltiptext) property to all navigation nodes where it is not set explicitly. 
```

Thank you for adding more docu. But we probably shouldn't call it a "dumb mechanism" 😄 .

> - **default**: the parameter **defaults.tooltipText** is taking from navigation config, and you can overwrite it using the **tooltipText** value on a single node level.

Reviewer: ```suggestion
- **default**: the parameter **defaults.tooltipText** is `false` by default, and you can overwrite it using the **tooltipText** value on a single node level.
```

I guess according to [this comment](https://github.com/SAP/luigi/pull/2242#pullrequestreview-744075750) false is the default, i.e. no tooltip will be shown unless explicitly specified.

> - **description**: applies the [tooltipText](navigation-parameters-reference.md#tooltiptext) property to all navigation nodes where it is not set explicitly. If **tooltipText** is added to some specific node but it set value as null or undefined, user should see value frome **defaults.tooltipText**.

Reviewer 2: ```suggestion
- **description**: applies the [tooltipText](navigation-parameters-reference.md#tooltiptext) property to all navigation nodes where it is not set explicitly. If **tooltipText** is added to some specific node and its is set as as `null` or `undefined`, the value of  **defaults.tooltipText** is used instead.
```
